external/llvm-project/mlir/lib/Transforms/Utils/OutlinerUtils.cpp

```diff
@@ -344,7 +353,14 @@ static void outlineOps(Operation *anchorOp, ArrayRef<Operation *> trailingOps,
   if (!trailingOps.empty())
     lastOp = trailingOps[trailingOps.size() - 1];
   b.setInsertionPointAfter(lastOp);
-  func::CallOp callOp = b.create<func::CallOp>(loc, outlinedFunc, values);
+  //   // +++pf: Move this bit up if we want fusedLoc on the FuncOp.
+  //   SmallVector<Location> fusedLocs{anchorOp->getLoc()};
+  //   for (auto *op : leadingOps)           // llvm::concat<Operation*> barfs
+  //     fusedLocs.push_back(op->getLoc());
+  //   for (auto *op : trailingOps)
+  //     fusedLocs.push_back(op->getLoc());
```

Comment:
It might make sense to say "this function came from pulling together all of these things", yeah.